src/java/ognl/OgnlRuntime.java

```diff
@@ -882,7 +883,7 @@ public static Object invokeMethod(Object target, Method method, Object[] argsArr
 
```

Comment:
Current 3.1.23 changes could be incorporated here.

Reply:
:confused: but I'm synced with latest 3.1 branch!

Reply:
Please ignore the comments about the "upper-block" differences (I was getting confused with some local changes in many back-and-forth copying) :blush:

For `invokeMethod`  changes in 3.1.23 vs. this PR, I think Git sees the "whole block" changing (so no conflict per se).  Performing a manual compare it looks like the new logic in this PR's invokeMethod uses the original 3.1.22 logic (so it would overwrite the usage of `methodAccessCacheValue`, `methodPermCacheValue` in 3.1.23).  At least I think so ? :smiley:

I _just noticed_ that the `@since` blocks in the PR are at `3.1.23`, so they could be changed to `3.1.24` too ?

Reply:
Oh sorry you were right. There is a recent release. Thanks!